cleansweep/core/mailer.py

```diff
@@ -47,9 +46,6 @@ def sendmail(to_address, subject, message, message_html=None, reply_to=None, cc=
     if reply_to:
         headers['Reply-To'] = reply_to
 
-    if message_html:
-        message_html = pynliner.fromString(message_html)
-
```

Comment:
The editor does that already. It returns the source code with inline css. That's why I removed it.